Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/PrimerRawOTPDataTokenizationBuilder.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  PrimerRawOTPDataTokenizationBuilder.swift
+//  PrimerSDK
+//
+//  Created by Boris on 26.9.24..
+//
+
+// swiftlint:disable function_body_length
+
+import Foundation
+
+class PrimerRawOTPDataTokenizationBuilder: PrimerRawDataTokenizationBuilderProtocol {
+
+    var rawData: PrimerRawData? {
+        didSet {
+            if let rawPhoneNumberInput = self.rawData as? PrimerOTPData {
```

Comment:
This variable name is a little misleading

Reply:
Fixed and pushed.